packages/flame/analysis_options.yaml

```diff
@@ -1 +1,5 @@
-include: package:flame_lint/analysis_options_with_dcm.yaml
\ No newline at end of file
+include: package:flame_lint/analysis_options_with_dcm.yaml
+
+analyzer:
+  errors:
+    todo: ignore
```

Comment:
do we need to "ignore" todos? I was pretty sure we allowed them, we have several all over the place

Reply:
Ooosie

Reply:
Ooosie